docs/tooling.md

```diff
@@ -25,17 +25,19 @@ In particular:
 
 - `leafwing-input-manager` and `bevy_mod_picking` are very likely to be upstreamed into Bevy in the near future.
 - `bevy-inspector-egui` and `bevy_mod_debugdump` help fill the gap until Bevy has its own editor.
-- `bevy_rapier` or `avian` helps fill the gap until Bevy has its own physics engine.
+- `avian` or `bevy_rapier` helps fill the gap until Bevy has its own physics engine. `avian` is easier to use, `bevy_rapier` has better performance.
 - `sickle_ui` is well-aligned with `bevy_ui` and helps fill the gap until Bevy has a full collection of UI widgets.
 
+None of these are necessary, but they can save you a lot of time and effort.
```

Comment:
Redundant?

Reply:
Some people are a bit allergic to third party deps out of principle in my experience, so I'd like to stress it.

Reply:
Understandable.